libs/components/forms/src/lib/modules/selection-box/selection-box-adapter.service.ts

```diff
@@ -74,7 +74,7 @@ export class SkySelectionBoxAdapterService {
    * Returns the width of the `parentNode` of the provided `element`.
    */
   public getParentWidth(element: ElementRef): number {
-    return element.nativeElement.parentNode.getBoundingClientRect().width;
+    return element.nativeElement?.parentNode?.getBoundingClientRect()?.width;
```

Comment:
There needs to be `| undefined` on the return type or `|| 0` on the return statement.

Reply:
How did strict mode not catch this?

Reply:
Because of typing.... `ElementRef`'s `nativeElement` is `any` and so everything downstream is `any`.

Reply:
I added some typing here so that it would have been caught.